src/client/App.js

```diff
@@ -43,8 +43,8 @@ function App() {
             <Route exact path="/selection">
               <SelectionPage />
             </Route>
-            <Route exact path="/registration/student">
-              <StudentRegistrationPage />
+            <Route exact path="/registration/admin">
+              <AdminRegistrationPage />
```

Comment:
@Deepti-Dk You removed the previous student registration. You should have both, right?

Reply:
yes, added again